scripts/insert_into_appointments.sql

```diff
@@ -1 +1,7 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(given_name, surname, doctor_id, datetime)
+  VALUES ('Marsha', 'Wilcox', 4, '2016-08-12 10:00:00'),
+         ('Ivory', 'Talley', 6 , '2016-08-17 13:00:00'),
+         ('Marsha', 'Wilcox', 6,'2016-08-12 10:00:00'),
+         ('Blake', 'Hodge', 4, '2016-08-17 13:00:00')
+;
```

Comment:
The `appointments` table doesn't have `given_name` and `surname` columns for patient information; that information is in the `patients` table. You should have set it up with `patients` like you did with `doctors`.